database/config.js

```diff
@@ -24,7 +24,7 @@ const config = {
     ssl: process.env.HEROKU_SSL
   }
 }
-const client = new pg.Client(config.local);
+const client = new pg.Client(config.heroku);
```

Comment:
this switches around a lot, so lets make it an env variable, something like...

    const env = process.env.APP_CONFIG || 'local';
    const client = new pg.Client(config[env]])